src/main/java/study/racingcar/domain/Position.java

```diff
@@ -0,0 +1,31 @@
+package study.racingcar.domain;
+
+import java.util.Objects;
+
+public class Position {
+    private int position;
+
+    public Position(int position) {
+        this.position = position;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public int add(int value) {
+        this.position = this.position + value;
+        return this.position;
```

Comment:
`Position`을 VALUE OBJECT로 관리하면 어떨까요?

```suggestion
        return new Position(this.position + value);
```

Reply:
리턴값을 사용하는 곳이 없어서 리턴 타입을 `void`로 바꾸었습니다. 😺